rabbitMQ/ReceivingMessage.js

```diff
@@ -0,0 +1,37 @@
+import amqp from 'amqplib';
+import addWebsite from '../controllers/website.js';
+import backupSite from '../Server.js';
+import logger from '../logger.js';
+
+export const handleData = async (data) => {
+  const functionName = data.extraParam;
+  switch (functionName) {
+  case 'addWebsite':
+    addWebsite(data);
+    break;
+  case 'backupSite':
+
+    backupSite(data);
+    break;
+    // case 'addBackup':
+    //   addBackup(data);
+    //   break;
+  default:
+    logger.info(`Function ${functionName} not found.`);
+  }
+};
+
+export const startConsumer = async () => {
+  const connection = await amqp.connect('amqp://localhost');
+  const channel = await connection.createChannel();
+  const queueName = 'website-subscriptions';
```

Comment:
save this value in a config file, and use it from there, here and in send_message file.